pkg/proxy/iptables/proxier.go

```diff
@@ -1081,7 +1081,12 @@ func (proxier *Proxier) syncProxyRules() {
 					// Jump to the service chain.
 					writeLine(proxier.natRules, append(args, "-j", string(svcChain))...)
 				} else {
-					// TODO: Make all nodePorts jump to the firewall chain.
+					for _, lp := range lps {
+						writeLine(proxier.natRules,
```

Comment:
It's not clear to me why this is ONLY in the non NodeLocal path -- shouldn't it be in both?  The LB IP logic embeds the OnlyLocal decision into the fwChain